README.md

```diff
@@ -1 +1,108 @@
 # shinycoreci-apps
+
+Test applications and workflows for Shiny R packages.
+
+There are three main workflows:
+* [**shinytest:**](https://github.com/rstudio/shinycoreci-apps/actions?query=workflow%3Ashinytest) Test applications using [`rstudio/shinytest`](https://github.com/rstudio/shinytest)
+* [**shinyjster:**](https://github.com/rstudio/shinycoreci-apps/actions?query=workflow%3Ashinyjster) Test applications using [`schloerke/shinyjster`](https://github.com/schloerke/shinyjster)
+* [**testthat**](https://github.com/rstudio/shinycoreci-apps/actions?query=workflow%3Atestthat): Test applications using [`rstudio/testthat`](https://github.com/rstudio/testthat) via INTEGRATION_TESTING_LINK
+
+
+## Usage
+
+```r
+# (Install `shinycoreci` once per R session)
+remotes::install_github("rstudio/shinycoreci")
+
+# Sitting at the root folder of the rstudio/shinycoreci-apps repo
+shinycoreci::test_shinytest()
+shinycoreci::test_shinyjster()
+shinycoreci::test_testthat()
+```
+
+This GitHub repo works in concert with [`rstudio/shinycoreci`](https://github.com/rstudio/shinycoreci)
+
+Each of the major `test_*()` methods will (by default) ask a [Spreadsheet of Apps](https://docs.google.com/spreadsheets/d/1jPWPNmSQbbE8E6KS5tXnm5Jq7r01GaOCCE1Vvz5e9a8/edit#gid=0) to determine which apps to test.  `test_testthat()` will inspect all apps in the supplied directory inspecting for a `tests/testthat.R` file.
+
+New apps **must be added** to the [spreadsheet](https://docs.google.com/spreadsheets/d/1jPWPNmSQbbE8E6KS5tXnm5Jq7r01GaOCCE1Vvz5e9a8/edit#gid=0).  Check the appropriate boxes to trigger respective workflows.
+
+
+### Diagnosing `shinytest` problems posted by GitHub Actions
+
+```r
+# Sitting at the root folder of the rstudio/shinycoreci-apps repo
+
+# git pull
+# git checkout BROKEN_BRANCH_NAME
```

Comment:
It would help to say how someone is supposed to find the broken branch, for example, with a screenshot:
<img width="813" alt="Screen Shot 2020-02-14 at 2 42 28 PM" src="https://user-images.githubusercontent.com/86978/74566085-46a1e780-4f38-11ea-97b1-bd6b992e6a5d.png">